Makefile

```diff
@@ -4,27 +4,31 @@ PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 UNITDIR ?= $(PREFIX)/lib/systemd/system
-DSSLIBDIR ?= $(PREFIX)/lib/docker-storage-setup
+CSSLIBDIR ?= $(PREFIX)/lib/container-storage-setup
 SYSCONFDIR ?= $(DESTDIR)/etc/sysconfig
 
 .PHONY: clean
 clean:
 	-rm -rf *~ \#* .#*
 
 .PHONY: install
-install: install-docker
+install: install-docker install-core
 
 .PHONY: install-docker
-install-docker: install-core
-	install -D -m 644 ${DOCKER}-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
+install-docker:
+	install -D -m 644 docker-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
```

Comment:
If a user overrides DOCKER it will blow up.  There is only one docker-storage-setup.service in our pool,  If you wanted to install 
make DOCKER=docker-latest install

This would blow up.